### ISSUE TITLE
[dex] toggle portfolio balances visibility

### ISSUE THREAD

**Issue by fredev** [lira-dao/ecosystem#56]
A function to hide the balances and USD value of assets should be implemented in the portfolio page

- implement an eye icon that allows switching between visible and hidden values
- when hidden values ​​should be replaced with asterisks
- the eye icon should be crossed out when the values ​​are visible and normal when the values ​​are hidden

Reward: 200 LDT

**Comment by ManyRios** [comment]
Hi! what is LDT?

**Comment by Oyase-shinobi** [comment]
Hi, I can finish this issue
plz let me know

**Comment by fredev** [comment]
sorry for the late response, I've totally missed the comments

@ManyRios LDT is one of the tokens that are part of the lira dao ecosystem, head over liradao.org and you can find links to whitepaper and discord community

@Oyase-shinobi please join our discord and open a support ticket, we have a dedicated channel for developers

**Comment by Oyase-shinobi** [comment]
> sorry for the late response, I've totally missed the comments
> 
> @ManyRios LDT is one of the tokens that are part of the lira dao ecosystem, head over liradao.org and you can find links to whitepaper and discord community
> 
> @Oyase-shinobi please join our discord and open a support ticket, we have a dedicated channel for developers

Thank you for reaching out me
Could you share your discord ID and server for better communication?

**Comment by nicholas1990** [comment]
Hi!
https://discord.gg/aEhma3wq

**Comment by Oyase-shinobi** [comment]
Hi @nicholas1990 
are you team member?

**Comment by nicholas1990** [comment]
Yes, I'm the co-founder of the Lira DAO project and a Full Stack developer. Feel free to join our Discord and open a support ticket to speak directly with the team.

**Comment by Oyase-shinobi** [comment]
> Yes, I'm the co-founder of the Lira DAO project and a Full Stack developer. Feel free to join our Discord and open a support ticket to speak directly with the team.

Thanks
Could you check your dm in discord?

**Comment by fredev** [comment]
@Oyase-shinobi please do not dm team members, open a support ticket
